src/add.test.js

```diff
@@ -0,0 +1,18 @@
+const addToDo = require('./add.js');
+
+const oldTask = [
+  { description: 'Task1', completed: false, index: 1 },
+  { description: 'Task2', completed: false, index: 2 },
+];
+
+const AddedTask = [
+  { description: 'Task1', completed: false, index: 1 },
+  { description: 'Task2', completed: false, index: 2 },
+  { description: 'newtask', completed: false, index: 3 },
+];
+
+test('Add a newtask', () => {
+  expect(
+    addToDo(oldTask, 'newtask'),
+  ).toEqual(AddedTask);
+});
```

Comment:
- Kindly, Make sure you group your tests using the describe() method because using describe methods will describe your test clearly 👍👍